### PR TITLE
Update profile picture

### LINKED ISSUE #15: As a user I want to edit my profile picture and name
8H of estimation

### PR DESCRIPTION
**Pull request by fmadueno**

### CODE REVIEW

File: backend/src/repositories/user-repository.js [hunk 1]
```diff
@@ -1,57 +1,64 @@
...
+}
+
+const findById = async (id) => {
+    return userModel.findById(id).populate("lists", "title")
+}
+
+//TODO remove this method
+const findAll = async (id) => {
+    return userModel.find();
+}
```
Comment (vimtor) [inline backend/src/repositories/user-repository.js]: Remove this method

File: backend/src/routers/list-router.js [hunk 2]
```diff
@@ -11,9 +11,10 @@ router.get('/', async (req, res) => {
     res.json(lists)
 })
 
+//TODO - Remove?
 router.get('/:id', async (req, res) => {
-    const list = await listService.getById(req.params.id)
-    res.json(list)
+    const lists = await listService.getListsFromOwner(req.userId)
+    res.json(lists)
```
Comment (vimtor) [inline backend/src/routers/list-router.js]: Go back to old version

File: backend/src/routers/user-router.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    res.json(users)
+})
+
+//TODO remove this method?
+router.get('/', async (req, res) => {
+    const users = await userService.getAllUsers()
+    res.json(users)
+})
+
+export default router
```
Comment (vimtor) [inline backend/src/routers/user-router.js]: Remove not used methods

File: backend/src/services/user-service.js [hunk 4]
```diff
@@ -1,37 +1,55 @@
...
+
+//TODO remove this function?
+const getUser = async (userId) => {
+    return userRepository.findById(userId);
+}
+
+//TODO remove this function?
+const getAllUsers = async () => {
+    return userRepository.findAll();
+}
```
Comment (vimtor) [inline backend/src/services/user-service.js]: Same thing

File: frontend/src/components/profile-display/profile-display.js [hunk 5]
```diff
@@ -1,25 +1,49 @@
...
+Modal.setAppElement('#root')
 
 const ProfileDisplay = () => {
     const history = useHistory()
+    const {activeUser, alias, avatar, updateProfile} = useUsers()
     const {logout} = useAuth()
+    const [isOpen, setIsOpen] = useState(false);
+
+    let avatar64 = [];
+    if (avatar) avatar64 = Buffer.from(avatar);
```
Comment (vimtor) [inline frontend/src/components/profile-display/profile-display.js]: I think is better doing:

```js
const avatar64 = useMemo(() => avatar ? Buffer.from(avatar) : [], [avatar])
```

**Review by vimtor — CHANGES_REQUESTED** [review]
Pretty good overall 💋

I requested some changes

File: backend/src/repositories/user-repository.js (hunk 1 above)
Comment (fmadueno) [reply backend/src/repositories/user-repository.js]: Removed.

File: backend/src/routers/user-router.js (hunk 3 above)
Comment (fmadueno) [reply backend/src/routers/user-router.js]: Removed.

**Review by vimtor — APPROVED** [review]

**Pull request merged**